webpack.config.js

```diff
@@ -35,9 +33,10 @@ module.exports = {
         output: {
           ascii_only: true
         }
-      },
-      sourceMap: true
+      }
     })]
   },
   mode: 'production'
```

Comment:
Note: you have `mode: 'production'` here, but `mode: 'development'` earlier on.  Do you mean to include both?

Reply:
Thanks. Good find.